uint8 team_number
uint8 team_color
uint8 score
uint8 penalty_shot
uint16 single_shots
uint8 coach_sequence
string coach_message
gamecontroller_msgs/RobotInfo coach
gamecontroller_msgs/RobotInfo[] players